projects/observability/src/shared/components/log-events/log-events-table.component.test.ts

```diff
@@ -51,7 +51,7 @@ describe('LogEventsTableComponent', () => {
         id: 'summary'
       })
     ]);
-    expect(spectator.query(TableComponent)!.pageable).toBe(true);
+    expect(spectator.query(TableComponent)!.pageable).toBe(false);
```

Comment:
can't we just remove this line altogether? Default state of `Pageable` is false, right?

Reply:
oh yes, and that way they won't miss testing it. Okay cool